dipy/workflows/io.py

```diff
@@ -27,13 +27,11 @@ def run(self, input_files,
         input_files : variable string
             Any number of Nifti1, bvals or bvecs files.
         b0_threshold : float, optional
-            (default 50)
```

Comment:
The explanation of the parameter is missing.

Reply:
Addressed in ce1f6b1